backend/app/routers/auth.py

```
@@ -108,3 +108,32 @@
+            full_name=user_info["full_name"],
+            google_id=user_info["google_id"],
+            is_verified=user_info["email_verified"],
+            role="user",
+            password=None,
+        )
+
+        db.add(new_user)
+        db.commit()
+        db.refresh(new_user)
+        access_token = create_access_token({"sub": new_user.email})
+        return {"access_token": access_token, "token_type": "bearer"}
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)